packages/e2e-tests/puppeteer/scripts/web3torrent.ts

```diff
@@ -31,16 +31,21 @@ export async function uploadFile(
   metamask: Dappeteer,
   filePath?: string
 ): Promise<string> {
-  await page.waitForSelector('input[type=file]');
-
+  // https://pub.dev/documentation/puppeteer/latest/puppeteer/FileChooser-class.html
+  // Not clear why puppeteer FileChooser won't work out of box. We are doing it manually for now.
+  const inputUploadHandle = await page.waitForSelector('input[type=file]');
+  if (inputUploadHandle.getProperty('disabled')) {
+    console.warn('Input button is disabled!!');
+    await page.waitFor(4000);
```

Comment:
Couldn't we use `[disabled=false]` in the selector to wait for it being enabled?

Reply:
Maybe, I will try. It might have to be `disabled=undefined`

Reply:
6a4086a

Reply:
Tested with https://chrome.google.com/webstore/detail/css-selector-tester/bbklnaodgoocmcdejoalmbjihhdkbfon